wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractWMultiSelectList.java

```diff
@@ -198,10 +195,7 @@ protected void preparePaintComponent(final Request request) {
 	}
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this inheritDoc tag as the inheritance needs to be explicit due to the other comments in the block.